tasks/WhatisThat.tex

```diff
@@ -0,0 +1,64 @@
+\section{What is That?}
+\label{test:what-is-that}
+A guest asks the robot to identify objects in the household.
+
+\noindent \textbf{Focus:} \SysI{}, \HRI{}, \GestRec{}, \CV{}, \OR{}.
+
+
+\subsection*{Main Goal}
+The robot names 5 objects, the operator is pointing at.
+
+\noindent\textbf{Reward:} 1000pts (200pts per object)
+
+
+\subsection*{Bonus Rewards}
+\begin{enumerate}[nosep]
+	\item The robot understands two different non-verbal commands, e.g., nodding, gesturing for the robot to follow (250pts)
+	\item The robot names one feature that is unique to the object in its group (100pts, each)
+\end{enumerate}
+
+
+\subsection*{Setup and Procedure}
+\begin{enumerate}[nosep]
+	\item \textbf{Location:} This test takes place inside the \Arena{}.
+
+	\item \textbf{Groups of Objects:} There are five groups of 2--5 \KnownObjects{} randomly placed on the floor.
+	
+	\item \textbf{Guest:} The guest is one of the referees. They will step in front of the robot after it enters the \Arena{} and guide them to a group of objects where they point at one.
```

Comment:
This sentence is a bit confusing because of the bad grammar. Does the guest choose a group of objects and guide the robot, or does the robot guide the guest? I guess it should be the former?

Reply:
Hopefully, made it clearer